connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -656,6 +654,11 @@ object SparkSession extends Logging {
       this
     }
 
+    def interceptor(interceptor: ClientInterceptor): Builder = {
```

Comment:
doc please :)

Reply:
This is the public facing part of the interface and needs the most extensive documentation,.

Reply:
Ok, good point :). I didn't do it because other public methods here are undocumented, but that's probably not a good idea either